code_select.py

```diff
@@ -43,6 +43,8 @@ def main():
     code_text = parse_text_content(element)
     if PYPERCLIP:
         pyperclip.copy(code_text)
+        re_remove_dollars = re.compile(r"^(\$ )", re.MULTILINE)
+        pyperclip.copy(re.sub(re_remove_dollars, '', pyperclip.paste()))
```

Comment:
Can we move this line and above to line 43 and change it to `code_text = re.sub(re_remove_dollars, "", parse_text_content(element))`? That way it spans both conditionals and I don't think there is a need for `pyperclip.paste()`